swingset/src/main/java/com/nqadmin/swingset/SSBaseComboBox.java

```diff
@@ -332,6 +401,144 @@ public boolean hasSelection() {
 		Object item = getSelectedItem();
 		return item != null && item != nullItem;
 	}
+	
+	/**
+	 * {@inheritDoc }
+	 * This method override additionally selects a currently edited item in the combo box.
+	 * 
+	 * If there is a string in the combo editor that does not have any matches, it will revert the string.
+	 */
+	// **** GL STRICT/CONTAINS ****
+	//
+	// IT IS POSSIBLE THAT NO OVERRIDE OF setSelectedItem() WILL BE NEEDED IF GLAZEDLISTS FULLY SUPPORTS
+	// STRICT/CONTAINS
+	@Override
+	public void setSelectedItem(final Object _value) {
+		// 2020-12-30_BP: setSelectedItem outcomes:
```

Comment:
Probably not. I actually have an override of setEditable() to force it to false and block the developer, but it doesn't play well with GL so commented out for now. I'm not sure how this would work with Options/Mappings, but we can explore further by email/Issue you like.